tests/test-main.js

```diff
@@ -0,0 +1,24 @@
+var tests = [];
+for (var file in window.__karma__.files) {
+  if (window.__karma__.files.hasOwnProperty(file)) {
+    if (/test-.*?\.js$/.test(file) && !/test-main\.js/.test(file)) {
+      tests.push(file);
+    }
+  }
+}
+
+requirejs.config({
+  // Karma serves files from '/base'
+  baseUrl: '/base/lib/fxpay/',
+
+  paths: {
+    'helper': '../../tests/helper',
+    'es6-promise': '/base/lib/bower_components/es6-promise/promise',
```

Comment:
There's something else causing the travis build to barf (despite it working fine locally). I'll fix that first and then see about reducing `baseUrl` to `/base/lib` which should mitigate the need for the path for es6-promise.